apps/ex_fleet_yards_import/lib/ex_fleet_yards_import/importer/paint.ex

```diff
@@ -1,10 +1,186 @@
 defmodule ExFleetYardsImport.Importer.Paint do
   @moduledoc "Importer for paints"
 
+  require Logger
+
+  alias ExFleetYards.Repo.Game
+  import Ecto.Query
+
   use ExFleetYardsImport, data_source: :web_rsi, data_name: :paints
 
   @impl ExFleetYardsImport
+
   def import_data(_opts) do
-    {:error, :not_implemented}
+    items =
+      load_items()
+      |> Enum.map(&resolve_model/1)
+      |> Enum.filter(&filter_packs/1)
+      |> Enum.filter(&filter_without_models/1)
+
+    {:ok, items}
+  end
+
+  defp resolve_model(item) do
+    {model_name, paint_name} = extract_name(item["name"])
+
+    mapping = model_name_mapping(model_name)
+
+    models = load_models(model_name, mapping)
+
+    %{
+      name: paint_name,
+      model_name: model_name,
+      models: models,
+      mapping: mapping,
+      rsi_name: item["name"],
+      image: String.replace(item["media"]["thumbnail"]["storeSmall"], "store_small", "source"),
+      price: item["nativePrice"]["amount"],
+      discountedPrice: item["nativePrice"]["discounted"]
+    }
+  end
+
+  defp extract_name(paint_shop_name) do
+    captures = Regex.named_captures(~r/(?<model>.+)\s+-\s+(?<paint>.+)/, paint_shop_name)
+
+    if captures do
+      {String.trim(captures["model"]), String.trim(captures["paint"])}
+    else
+      {nil, paint_shop_name}
+    end
+  end
+
+  defp load_models(name, mapping)
+
+  defp load_models(nil, _) do
+    []
+  end
+
+  defp load_models(name, mapping) do
+    from(m in Game.Model,
```

Comment:
Could be moved to `ExFleetYards.Repo.Game.Model` as function there, but also fine here.